draft-ietf-quic-recovery.md

```diff
@@ -439,6 +456,13 @@ Either packet indicates that the Initial was received but not processed.
 Neither packet can be treated as an acknowledgment for the Initial, but they MAY
 be used to improve the RTT estimate.
 
+#### Discarding Initial State
+
+As described in Section 4.10 of {{QUIC-TLS}}, endpoints stop sending and
+receiving Initial packets once they start exchanging Handshake packets.  At this
+point, all loss recovery state for the Initial encryption level is also
+discarded.
```

Comment:
I'm not sure I understand what this mean; it seems like there is one set of variables. Do you mean you reset all the variables to the initial values.